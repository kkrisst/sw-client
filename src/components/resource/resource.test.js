import React from 'react'
import { shallow } from 'enzyme'

import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'

import Resource from './resource.jsx'

describe('Testing the Resource component', () => {

  const initialState = {};
  const mockStore = configureStore();


  it('expect to render Resource component', () => {
    const store = mockStore(initialState);

    
    expect(shallow(<Provider store={store}><Resource /></Provider>)).toMatchSnapshot()
  })
})
