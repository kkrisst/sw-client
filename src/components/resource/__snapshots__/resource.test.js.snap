// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Resource component expect to render Resource component 1`] = `ShallowWrapper {}`;
